packages/@okta/vuepress-site/docs/reference/api/org/index.md

```diff
@@ -0,0 +1,1248 @@
+{
+    "showEndUserFooter": true,
+    "_links": {
+        "hideEndUserFooter": {
+            "href": "https://${yourOktaDomain}.com/api/v1/org/preferences/hideEndUserFooter",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Show End User Page Footer
+<ApiOperation method="post" url="/api/v1/org/preferences/showEndUserFooter" />
+
+Show footer for End User Page
+
+#### Request path parameters
+
+N/A
+      
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+Fetched [Org Preferences](#org-preferences-object)
+
+#### Usage Examples
+
+The following request would show the footer for the end user page. 
+
+##### Request 
+```
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/preferences/showEndUserFooter"
+```
+
+##### Response 
+
+```json
+{
+    "showEndUserFooter": true,
+    "_links": {
+        "hideEndUserFooter": {
+            "href": "https://${yourOktaDomain}.com/api/v1/org/preferences/hideEndUserFooter",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Hide End User Page Footer
+<ApiOperation method="post" url="/api/v1/org/preferences/hideEndUserFooter" />
+
+Hide footer for End User Page
+
+#### Request path parameters
+
+N/A
+      
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+Fetched [Org Preferences](#org-preferences-object)
+
+#### Usage Examples
+
+The following request would hide the footer for the end user page. 
+
+#### Request 
+
+```
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/preferences/hideEndUserFooter"
+```
+
+#### Response
+
+```json
+{
+    "showEndUserFooter": false,
+    "_links": {
+        "hideEndUserFooter": {
+            "href": "https://${yourOktaDomain}.com/api/v1/org/preferences/showEndUserFooter",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Org API Objects
+
+### Org Setting object
+
+#### Org Setting properties
+
+The Org Setting object defines several properties:
+
+| Property              | Type                                                           | Description                           |
+|-----------------------|----------------------------------------------------------------|---------------------------------------|
+| _links                | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this object        |
+| address1              | String                                                         | Address1 of Org                       |
```

Comment:
I feel like this description (and the one below it) could use some elaboration.
```suggestion
| address1              | String                                                         | Address1 of org                       |
```

Reply:
Also lowercase "org" in all instances here. "Org" is only uppercase when referring to the object.